run.sh

```diff
@@ -2,12 +2,13 @@
 
 # Make a copy so we never alter the original
 cp -r /pkg /tmp/pkg
+cd /tmp/pkg
 
-# Make sure we can RW the copy
-chown -R notroot /tmp/pkg
-chmod -R u+rw /tmp/pkg
-find /tmp/pkg -type d -exec chmod u+x {} \;
+# Install (official repo + AUR) dependencies using yay. We avoid using makepkg
+# -s since it is unable to install AUR dependencies.
+yay -S --noconfirm $(pacman --deptest $(makepkg --printsrcinfo \
```

Comment:
It might be simpler to just run `./PKGBUILD && echo $deps $makedeps`, and that won't require `jq`. What do you think about that?

Reply:
Yeah I had thought about this. I've thought about this a little more now, initially I didn't want to do this to avoid execution of the PKGBUILD file, but makepkg is going to end up doing that anyway, so I think I think I will go ahead and switch it over to this